lib/src/chart/bar_chart/bar_chart_painter.dart

```diff
@@ -75,7 +75,7 @@ class BarChartPainter extends AxisChartPainter<BarChartData> with TouchHandler<B
       Size viewSize, List<BarChartGroupData> barGroups, BarChartAlignment alignment) {
     final Size drawSize = getChartUsableDrawSize(viewSize);
 
-    final List<double> groupsX = List<double>(barGroups.length);
+    final List<double> groupsX = List.filled(barGroups.length, 0, growable: false);

```

Comment:
Please revert this change, because I realized someone else changed this in a separate merge request #540.